Sources/Sugar/AnyCodable/AnyDecodable.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+
+/**
+ A type-erased `Decodable` value.
+
+ The `AnyDecodable` type forwards decoding responsibilities
+ to an underlying value, hiding its specific underlying type.
+
+ You can decode mixed-type values in dictionaries
+ and other collections that require `Decodable` conformance
+ by declaring their contained type to be `AnyDecodable`:
+
+     let json = """
+     {
+         "boolean": true,
+         "integer": 1,
+         "double": 3.14159265358979323846,
+         "string": "string",
+         "array": [1, 2, 3],
+         "nested": {
+             "a": "alpha",
+             "b": "bravo",
+             "c": "charlie"
+         }
+     }
+     """.data(using: .utf8)!
+
+     let decoder = JSONDecoder()
+     let dictionary = try! decoder.decode([String: AnyCodable].self, from: json)
+ */
+public struct AnyDecodable: Decodable {
+    public let value: Any
+
+    public init<T>(_ value: T?) {
+        self.value = value ?? ()
+    }
+}
+
+protocol _AnyDecodable {
+    var value: Any { get }
+    init<T>(_ value: T?)
```

Comment:
<!--
  1 failure:  Let and var shoul...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/Sugar/AnyCodable/AnyDecodable.swift;
  Line: 41;
-->
- :no_entry_sign: Let and var should be separated from other statements by a blank line. (`let_var_whitespace`)